index.html

```diff
@@ -0,0 +1,187 @@
+      <section class="sec1">
+        <p class="para1">
+          "Come One Come All"
+        </p>
+        <div class="title">
+          <h1 class="main-title">
+            <span>CULTURAL</span> MUSIC AFRICA CONCERT 2021
+          </h1>
+        </div>
+        <P class="para2">
+          Music is an essential element of culture. Sharing music from one culture to another gives people an insight into another way of life. All are invited from all countries in taking places in June, in Kenya
+        </P>
+        <h2 class=" year">
+          2021.06.20(FRI) ~ 21(SAT)
+        </h2>
+        <p class="para2 differ">
+          @Nyayo Stadium in Kenya, Cultural center and more
+        </p>
+      </section>
+      <section id="program" class="sec2">
+        <h2 class="sec-2-title">
+          Main Program
+        </h2>
+        <hr class="line">
+        <ul class="program">
+          <li class="list">
+            <i class="fas fa-archive"></i>
+            <h3 class="list-header">
+              Message
+            </h3>
+            <p class="list-para">
+              Listen and get the message from musicians from various countries about Culture and Music.
+            </p>
+          </li>
+          <li class="list">
+            <i class="fas fa-clipboard-list"></i>
+            <h3 class="list-header">
+              MC Exhibition
+            </h3>
+            <p class="list-para">
+              Appreciate various creations applying MC licence of Musicians, Culture Center. 
+            </p>
+          </li>
+          <li class="list">
+            <i class="fas fa-mail-bulk"></i>
+            <h3 class="list-header">
+              Forum
+            </h3>
+            <p class="list-para">
+              Have time to share your taughts and opinions with expert on each Song
+              of specific Culture.
+            </p>
+          </li>
+          <li class="list">
+            <i class="fas fa-edit"></i>
+            <h3 class="list-header">
+              Workshop
+            </h3>
+            <p class="list-para">
+              Try to create your own Songs using open source license rather than just watching the Musicians
+            </p>
+          </li>
+          <li class="list">
+            <i class="far fa-sun"></i>
+            <h3 class="list-header">
+              CC Ignite
+            </h3>
+            <p class="list-para">
+              Get oppotunities to work with MC affliates around the Africa also after the Concert get to know all.
+            </p>
+          </li>
+        </ul>
+        <p class="parr"><a href="#">SEE THE WHOLE PROGRAM</a>
+        </p>
+        <div class="join">
+          <h3>
+            Join MC Music Culture 2021
+          </h3>
+        </div>
+      </section>
+      <section id="guests" class="sec3">
+        <h2 class="sec-2-title">
+          Guest of Honour
+        </h2>
+        <hr class="line">
+        <ul class="speakers">
+
+        </ul>
```

Comment:
- [ ] Please add padding to the left and right to make the display of the speakers look like that of the original design.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/192044634-b9355503-1684-422f-9ff9-a41c3e7990fd.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/192044676-cf0b20d5-3cc9-41d2-9dfd-0082fa643d44.png)